src/plugins/discover/public/application/main/components/chart/discover_chart.tsx

```diff
@@ -18,16 +18,18 @@ import {
 import { i18n } from '@kbn/i18n';
 import type { DataView } from '@kbn/data-views-plugin/public';
 import { SavedSearch } from '@kbn/saved-search-plugin/public';
+import {
+  getVisualizeInformation,
+  triggerVisualizeActions,
+} from '@kbn/unified-field-list-plugin/public';
```

Comment:
@davismcphee In this PR I extracted `<FieldVisualizeButton />` to unified-field-list and moved these functions inside unified-field-list-plugin too. I guess you would also need them in unified-histogram, right? What could be a better solution for organising it?

Reply:
Good question, I haven't got to these yet in my refactoring so I haven't thought much about it, but I'll look into it and let you know what I think.